imsv-docs-astro/src/content/docs/guides/testing/passing-kyc-in-test.mdoc

```diff
@@ -0,0 +1,79 @@
+---
+title: Passing KYC in Test
+description: Instructions for how to navigate KYC in test to obtain a pass on the Immersve platform for each KYC mode.
+slug: guides/pass-kyc-in-testmode
+---
+
+When using test, there are procedures you can follow to ensure that you pass KYC checks in each mode.
```

Comment:
I think the key objectives of this guide are to explain how to provide test KYC data to consistently pass KYC in test mode. Could reword the intro to something like:
"This guide explains how to provide test KYC data to consistently pass KYC in testing, without the need to use real identity documents."

Reply:
yeah i was struggling with the wording for this, cause you have to use either our 1x fake document or a real one

Reply:
> This guide explains how to provide **test** KYC data to consistently pass KYC in **testing**, without the need to use real identity documents.
how important is it to reiterate test/testing? also see above re: you have to use a real doc or the 1 fake doc

Reply:
I think what you've written looks good 👍